openhub/management/commands/import_openhub_data.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+from django.core.management import call_command
+
+
+class Command(BaseCommand):
+	help = "This command invoke all the importing data command"
+	call_command('import_affiliated_committers_data')
```

Comment:
In Travis, the environment variables can not be exposed to Pull Requests, for security reasons.

As a result, this command needs to check if OH_TOKEN can be obtained.  If there is an exception, catch it, emit a note, and do nothing.